integration-tests/utils/integration-utils.js

```diff
@@ -61,6 +61,13 @@ const fillIn = async (page, xpath, text) => {
   return (await page.waitForXPath(xpath)).type(text, { delay: 20 })
 }
 
+// Replace pre-existing value
+const fillInReplace = async (page, xpath, text) => {
+  await (await findElement(page, xpath)).click({ clickCount: 3 }) // triple-click to replace the default text
+  await fillIn(page, xpath, text)
+  return await delay(1000) // Without this delay, the input field sometimes reverts back to its default value
```

Comment:
This `delay` is specific to the workflow page, right? If so, it shouldn't be baked into the utility. Otherwise, this util looks good.